### ISSUE TITLE
Sync Layer, Sync Canvas not working in Custom Workflow

### ISSUE THREAD

**Issue by Grant-CP** [AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin#447]
Hi, main summary is that I'm using a remote computer for comfyui and img2img workflows don't appear to be working well in the custom comfyui part of the extension.

Full issue is here: https://github.com/AbdullahAlfaraj/Comfy-Photoshop-SD/issues/11

Not sure where you wanted it to live. Let me know if there are any further tests and/or logs that you need other than the information that is in that other issue.

**Comment by abmmmm** [comment]
I also encountered this problem, very distressed

**Comment by abmmmm** [comment]
Restart PS, turn on real-time generation, the problem is no longer, do not know where the problem is

**Comment by Grant-CP** [comment]
Yeah you are totally right that worked. Honestly fine with me as the reason I wanted layer sync was for live generation anyway. I'm getting my directory filled up with a bunch of buffer images, but at least it is working!

Should I mark this issue as solved do you think? I'm not familiar with issue etiquette.
